spec/lib/api/1.1/skus-spec.js

```diff
@@ -16,7 +17,9 @@ describe('Http.Api.Skus', function () {
         ]);
         return helper.startServer([
             dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
-        ]);
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
Just a suggestion or reminder, your code will work as well, but this is not necessary, because we have put it into global variables, on-http/spec/helper.js will includes on-core/spec/helper.js, this is our design.